controls/tabview/configuration.md

```diff
@@ -54,25 +54,25 @@ The available options are: `Top`, `Bottom`, `Left` and `Right`.
 </telerik:RadTabView>
 ```
 
-## Scrolling in the TabView header
+## Scrolling in the TabView Header
 
 TabView control allows you to scroll through the tabs inside the TabView Header. 
 
 To enable the scrolling use the `IsHeaderScrollable`(`bool`)property. The default value is `True`.
 
 ![TabView Scrolling in the Header](images/tabview-scrolling.gif "Scrolling in the TabView Header")
 
-## Overlaid header area
+## Overlaid Header Area
 
-* `IsHeaderOverlaid`(`bool`)&mdashIndicates whether the header area is overlaid on top of the content area. 
+* `IsHeaderOverlaid`(`bool`)&mdash;Indicates whether the header area is overlaid on top of the content area. 
 
-When the `IsHeaderOverlaid` is set to `True`, the header area is centered and partially overlaps with one of the borders of the content area, based on the value of the `HeaderPosition` property. The padding of the content area is adjusted accordingly, to avoid overlapping with the header area.
+When the `IsHeaderOverlaid` is set to `True`, the header area is centered and  overlaps with one of the borders of the content area, based on the value of the `HeaderPosition` property. The padding of the content area is adjusted so, to avoid overlapping with the header area.
```

Comment:
```suggestion
When the `IsHeaderOverlaid` is set to `True`, the header area is centered and overlaps with one of the borders of the content area, based on the value of the `HeaderPosition` property. The padding of the content area is adjusted to avoid overlapping with the header area.
```